collect_app/src/main/java/org/odk/collect/android/tasks/DeleteInstances.kt

```diff
@@ -0,0 +1,49 @@
+package org.odk.collect.android.tasks
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import org.odk.collect.android.instancemanagement.InstanceDeleter
+import org.odk.collect.android.listeners.DeleteInstancesListener
+import timber.log.Timber
+
+class DeleteInstances(
```

Comment:
Please create a separate pr for this refactoring. This pr should be just about `ResetDialogPreferenceFragmentCompat`

Reply:
Done.